whatToDad_site/whatToDad_app/templates/activity.html

```diff
@@ -26,9 +26,52 @@ <h1> Activities </h1>
     <br>
     <form method="POST" action="{% url 'activity' %}">
         {% csrf_token %}
-        {{ form }}
-        <input type="button" class="btn btn-secondary" value="Cancel" data-dismiss="modal"></input>
-        <input type="submit" class="btn btn-primary" name="add" value="Add activity"></input>
+        <div class="row">
+            <div class="col">
+                <h3 class="form-control">Detail: {{ form.detail }}</h3>
+            </div>
+
+            <div class="col">
+                <h3 class="form-control">Min-age: {{ form.min_age }}</h3>
+            </div>
+
+            <div class="col">
+                <h3 class="form-control">Max-age: {{ form.max_age }}</h3>
```

Comment:
It's weird to me to have a max age without an option to have no limit. Very few people will face the restriction but if people think 100 sounds high enough some people may be over the max and get confused why they aren't allowed. I think that's unlikely but having a way to say there's no age limit would be cleaner for users. They aren't forced to make up their own reasonable max age.